Fixes colour of Playlist button

Alipoodle opened the pull request:
Unsure if this should be done in this file or full.css

This fixes the problem where the play buttons and the pulsing effect around it within the playlist bar isn't themed

This is in both Dark: http://puu.sh/rMn4T/40869933e8.png
And Light theme http://puu.sh/rMnu4/17faf13977.png

Alipoodle committed:
Fixes colour of Playlist button

Unsure if this should be done in this file or full.css

This fixes the problem where the play buttons and the pulsing effect around it within the playlist bar isn't themed

This is in both Dark: http://puu.sh/rMn4T/40869933e8.png
And Light theme http://puu.sh/rMnu4/17faf13977.png

chrismou commented:
Looks good to me :+1:

MarshallOfSound commented on lib/common.css:
```diff
@@ -197,3 +197,13 @@ button.goog-buttonset-default:hover {
+
+.material .sj-play-button-0 #pulse.sj-play-button {
+    background: <<FORE_SECONDARY>>;
+    opacity: .3;
```
This is simulating a different color right?  Instead of opacity we should probably use our `ALPHA` function so that the entire button doesn't become transparent, just the background color.

https://github.com/gmusic-utils/gmusic-theme.js/blob/master/src/gmusic-theme.js#L189

Alipoodle replied:
This is the little pulsing effect around...
Alternatively it can be changed to be ALPHA if you would prefer it:

`background: <<ALPHA 0.3>><<FORE_SECONDARY>><<ALPHA>>`

Alipoodle replied:
After Testing this it doesn't seem to work? http://puu.sh/rNbL9/3d54118a7f.png

Although i may have inputted it in wrong:
`.material .sj-play-button-0 #pulse.sj-play-button {
    background: <<ALPHA 0.3>><<FORE_SECONDARY>><<ALPHA>>;
}`

Alipoodle replied:
Forgot the / for the ending ALPHA, and works perfectly fine

jostrander replied:
@MarshallOfSound sort of unrelated, but isn't that regex incorrectly not escaping the `.`, which would end up matching anything in that space?

Alipoodle committed:
Changes to GPMDP Alpha

Changes from  to GPMDPs Aplha

Alipoodle committed:
Changes to GPMDP Alpha

Changes from opactiy to GPMDPs Alpha

Alipoodle commented:
Now using ALPHA instead of Opacity.

Alipoodle committed:
Increased Alpha

After testing for a bit i feel that, i need to change alpha up a little to about 0.5 to match
Using Alpha: http://puu.sh/rQWOu/098828ce49.png
Stock: http://puu.sh/rQWPl/5a31d3b860.png

Alipoodle commented:
😃
